test/unit.cpp

```diff
@@ -12740,7 +12742,8 @@ TEST_CASE("regression tests")
 
         CHECK(j2a.dump() == "23.42");
         //issue #230
-        //CHECK(j2b.dump() == "23.42");
+        CHECK(j2b.dump() == "2.342e01");
```

Comment:
❗ These two lines do not express the desired behavior: The idea of the adjustment was that dumping parsed numbers yield the same string as before. In the examples, string `""2342e-2""` should be dumped exactly as `"2342e-2"` and not `"2.342e01"`.

Likewise `j3b` should be dumped as `"10E3"`.